### PR TITLE
Update supported file formats

### PR DESCRIPTION
**Pull request by Jessie212**
## Description

This change updates property descriptions and examples with supported file formats for password authentication.

## Release notes

(x) This is not user-visible or docs only and no release notes are required.
( ) Release notes are required, please propose a release note for me.
( ) Release notes are required, with the following suggested text:

```markdown
# Section
* Fix some things. ({issue}`issuenumber`)
```

### CODE REVIEW

**Review by Praveen2112 — APPROVED** [review]

**Comment by Praveen2112** [comment]
There is a failure in docs suite. Can you PTAL ?

**Comment by hashhar** [comment]
Unrelated to the docs change but looks like something to investigate:

```
Error:  io.trino.tests.TestSpilledOrderByQueries.testOrderByWithNulls  Time elapsed: 1.01 s  <<< FAILURE!
java.lang.AssertionError: Execution of 'actual' query failed: SELECT orderkey, custkey, orderstatus FROM orders ORDER BY nullif(orderkey, 3) DESC NULLS LAST, custkey ASC
	at org.testng.Assert.fail(Assert.java:83)
	at io.trino.testing.QueryAssertions.assertQuery(QueryAssertions.java:150)
	at io.trino.testing.QueryAssertions.assertQuery(QueryAssertions.java:106)
	at io.trino.testing.AbstractTestQueryFramework.assertQueryOrdered(AbstractTestQueryFramework.java:381)
	at io.trino.testing.AbstractTestQueryFramework.assertQueryOrdered(AbstractTestQueryFramework.java:371)
	at io.trino.testing.AbstractTestQueryFramework.assertQueryOrdered(AbstractTestQueryFramework.java:366)
	at io.trino.testing.AbstractTestOrderByQueries.testOrderByWithNulls(AbstractTestOrderByQueries.java:150)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: io.trino.testing.QueryFailedException: Index (0) is greater than or equal to list size (0)
	at io.trino.testing.AbstractTestingTrinoClient.execute(AbstractTestingTrinoClient.java:122)
	at io.trino.testing.DistributedQueryRunner.execute(DistributedQueryRunner.java:489)
	at io.trino.testing.QueryAssertions.assertQuery(QueryAssertions.java:147)
	... 18 more
	Suppressed: java.lang.Exception: SQL: SELECT orderkey, custkey, orderstatus FROM orders ORDER BY nullif(orderkey, 3) DESC NULLS LAST, custkey ASC
		at io.trino.testing.DistributedQueryRunner.execute(DistributedQueryRunner.java:492)
		... 19 more
Caused by: java.lang.IndexOutOfBoundsException: Index (0) is greater than or equal to list size (0)
	at it.unimi.dsi.fastutil.longs.LongArrayList.getLong(LongArrayList.java:267)
	at io.trino.operator.PagesIndex.buildPage(PagesIndex.java:306)
	at io.trino.operator.PagesIndex$2.computeNext(PagesIndex.java:631)
	at io.trino.operator.PagesIndex$2.computeNext(PagesIndex.java:624)
	at com.google.common.collect.AbstractIterator.tryToComputeNext(AbstractIterator.java:146)
	at com.google.common.collect.AbstractIterator.hasNext(AbstractIterator.java:141)
	at io.trino.operator.WorkProcessorUtils.lambda$fromIterator$0(WorkProcessorUtils.java:103)
	at io.trino.operator.WorkProcessorUtils$ProcessWorkProcessor.process(WorkProcessorUtils.java:413)
	at io.trino.operator.WorkProcessorUtils$3.process(WorkProcessorUtils.java:347)
	at io.trino.operator.WorkProcessorUtils$ProcessWorkProcessor.process(WorkProcessorUtils.java:413)
	at io.trino.operator.WorkProcessorUtils$3.process(WorkProcessorUtils.java:347)
	at io.trino.operator.WorkProcessorUtils$ProcessWorkProcessor.process(WorkProcessorUtils.java:413)
	at io.trino.operator.WorkProcessorUtils$2.process(WorkProcessorUtils.java:126)
	at io.trino.operator.WorkProcessorUtils$ProcessWorkProcessor.process(WorkProcessorUtils.java:413)
	at io.trino.operator.WorkProcessorUtils.getNextState(WorkProcessorUtils.java:262)
	at io.trino.operator.WorkProcessorUtils$YieldingProcess.process(WorkProcessorUtils.java:182)
	at io.trino.operator.WorkProcessorUtils$ProcessWorkProcessor.process(WorkProcessorUtils.java:413)
	at io.trino.operator.WorkProcessorUtils$3.process(WorkProcessorUtils.java:347)
	at io.trino.operator.WorkProcessorUtils$ProcessWorkProcessor.process(WorkProcessorUtils.java:413)
	at io.trino.operator.WorkProcessorUtils$YieldingIterator.computeNext(WorkProcessorUtils.java:81)
	at io.trino.operator.WorkProcessorUtils$YieldingIterator.computeNext(WorkProcessorUtils.java:67)
	at com.google.common.collect.AbstractIterator.tryToComputeNext(AbstractIterator.java:146)
	at com.google.common.collect.AbstractIterator.hasNext(AbstractIterator.java:141)
	at io.trino.operator.OrderByOperator.getOutput(OrderByOperator.java:275)
	at io.trino.operator.Driver.processInternal(Driver.java:394)
	at io.trino.operator.Driver.lambda$process$8(Driver.java:297)
	at io.trino.operator.Driver.tryWithLock(Driver.java:689)
	at io.trino.operator.Driver.process(Driver.java:289)
	at io.trino.operator.Driver.processForDuration(Driver.java:260)
	at io.trino.execution.SqlTaskExecution$DriverSplitRunner.processFor(SqlTaskExecution.java:773)
	at io.trino.execution.executor.PrioritizedSplitRunner.process(PrioritizedSplitRunner.java:165)
	at io.trino.execution.executor.TaskExecutor$TaskRunner.run(TaskExecutor.java:523)
	at io.trino.$gen.Trino_testversion____20230308_051258_39925.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
```

**Comment by mosabua** [comment]
Ping me to chat about this @Jessie212

File: docs/src/main/sphinx/security/ldap.rst [hunk 1]
```diff
@@ -93,10 +93,18 @@ Property                           Description
                                    ``ldap.allow-insecure=true``.
 ``ldap.allow-insecure``            Allow using an LDAP connection that is not secured with
                                    TLS.
-``ldap.ssl.keystore.path``         Path to the PEM or JKS key store.
+``ldap.ssl.keystore.path``         Path to the PEM or the JKS keystore file. The
+                                   PEM file must be a text file with the Base64
+                                   encoded certificate. Common extenstions are
```
Comment (electrum) [inline docs/src/main/sphinx/security/ldap.rst:98]: Typo

File: docs/src/main/sphinx/security/ldap.rst [hunk 2]
```diff
@@ -93,10 +93,18 @@ Property                           Description
                                    ``ldap.allow-insecure=true``.
 ``ldap.allow-insecure``            Allow using an LDAP connection that is not secured with
                                    TLS.
-``ldap.ssl.keystore.path``         Path to the PEM or JKS key store.
+``ldap.ssl.keystore.path``         Path to the PEM or the JKS keystore file. The
+                                   PEM file must be a text file with the Base64
```
Comment (electrum) [inline docs/src/main/sphinx/security/ldap.rst:97]: This sentence doesn’t make sense to me. PEM *is* a text file with base64 encoded data (plus a header/footer). What’s important is the content of the base64.

We aren’t trying to describe PEM here, so we shouldn’t mention base64.

**Review by electrum — CHANGES_REQUESTED** [review]
The existing LDAP documentation matches  the other places in the documentation that configure keystore/truststore. If we want to explain PEM and JKS, that should be done in a central place. We might already have that and simply need a link from here.

**Review by colebow — APPROVED** [review]
LGTM if we're good on the technical content.

**Comment by mosabua** [comment]
Seems like the CI failure is a false alarm.

**Comment by mosabua** [comment]
@Jessie212  can you check if similar keystore and truststore properties also need that sort of link added as @electrum suggested?

**Review by mosabua — APPROVED** [review]
Looks good now. Thanks.

**Pull request merged**